examples/scala/build.sbt

```diff
@@ -169,3 +169,26 @@ lazy val root = (project in file("."))
     ),
     java17Settings
   )
+
+lazy val connect = (project in file("./connect"))
+  .settings(
+    run / fork := true,
+    name := "example",
+    scalaVersion := scala213,
+    mainClass := Some("example.DeltaConnect"),
+    libraryDependencies ++= Seq(
+      "io.delta" %% "delta-connect-client" % "4.0.0",
+      "org.apache.spark" %% "spark-connect-client-jvm" % "4.0.0"
+    ),
```

Comment:
same thing here, can you use `lookupSparkVersion` like we do in `getLibraryDependencies`? So we don't need to update versions manually

Reply:
(and use `getDeltaVersion` for the delta-connect version)